turbine/src/broadcast_stage/broadcast_fake_shreds_run.rs

```diff
@@ -45,6 +45,21 @@ impl BroadcastRun for BroadcastFakeShredsRun {
             .expect("Database error")
             .map(|meta| meta.consumed)
             .unwrap_or(0) as u32;
+        let chained_merkle_root = match next_shred_index.checked_sub(1) {
+            None => broadcast_utils::get_chained_merkle_root(
+                bank.slot(),
+                bank.parent_slot(),
+                blockstore,
+            )
+            .unwrap(),
+            Some(index) => {
+                let shred = blockstore
+                    .get_data_shred(bank.slot(), u64::from(index))
+                    .unwrap()
```

Comment:
Maybe use expect here?
```
.expect("Blockstore Error")
.expect("Shred not present")
```

Reply:
The 1st unwrap will print the `BlockstoreError` if it fails.
The 2nd unwrap will print `called Option::unwrap() on a None value`, indicating there was no shred.
Given that this code is only invoked in tests I think that is already sufficient here.